pkgs/development/python-modules/ax-platform/default.nix

```diff
@@ -66,6 +68,10 @@ buildPythonPackage rec {
     "--ignore=ax/service/tests/test_scheduler.py"
     "--ignore=ax/service/tests/test_with_db_settings_base.py"
     "--ignore=ax/storage"
+    # E   sqlalchemy.exc.ArgumentError: Type annotation for "SQAMetric.scalarized_objective_children_metrics"
+    # can't be correctly interpreted for Annotated Declarative Table form.
+    # (Background on this error at: https://sqlalche.me/e/20/zlpr)
+    "--ignore=ax/core/tests/test_experiment.py"
```

Comment:
This means that sqlalchemy 2+ is not supported, see #264724

Reply:
Thanks ! I wasn't sure.
I have removed the `sqlalchemy` dependency.